test/test-functions

```diff
@@ -442,8 +455,14 @@ install_config_files() {
     # set the hostname
     echo systemd-testsuite > $initdir/etc/hostname
     # fstab
+    if [[ "$LOOKS_LIKE_SUSE" ]]; then
+       ROOTMOUNT="/dev/sda1           /       ${FSTYPE}    rw 0 1"
+    else
+       ROOTMOUNT="LABEL=systemd           /       ${FSTYPE}    rw 0 1"
```

Comment:
The background ist that we intend to use the testsuite not only in the build environment but also in our QA. Therefore, I prefer to have the tests as close as possible to real life scenarios. Surely SUSE supports mount by label, but the default is mount by UUID. Mount by the device  also has some usage. We'd like to focus on the most common use cases for systemd testing and I saw mount by label failing in some older test setups. Therefore I try to avoid an unnecessary source of errors.